tools/jenkins/include/scripts/build-Linux-sdk.sh

```diff
@@ -3546,7 +3550,7 @@ fi
 echo
 echo "Check for broken libraries and binaries... (everything is OK if nothing is printed)"
 # SDK_HOME=/opt/Natron-sdk; LD_LIBRARY_PATH=$SDK_HOME/qt4/lib:$SDK_HOME/gcc/lib:$SDK_HOME/lib:$SDK_HOME/qt5/lib
-for subdir in . ffmpeg-gpl2 ffmpeg-lgpl libraw-gpl2 libraw-lgpl qt4 qt5; do
+for subdir in . ffmpeg-gpl2 ffmpeg-lgpl libraw-gpl2 libraw-lgpl qt4; do # qt5; do
```

Comment:
no need to remove qt5 from this line, it doesn't hurt

Reply:
Actually it does error out when missing the qt5 lib.